### PR TITLE
reworked authorizer

### PR DESCRIPTION
**Pull request by Dmdv**
### Changes
-

### Fixes
-

### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/gosdk/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- blobber:
- 0chain:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: zcnbridge/transaction/txn.go [hunk 1]
```diff
@@ -22,6 +24,30 @@ type (
 	}
 )
 
+type (
+	verifyOutput struct {
+		Confirmation confirmation `json:"confirmation"`
+	}
+
+	// confirmation represents the acceptance that a transaction is included into the blockchain.
+	confirmation struct {
```
Comment (dabasov) [inline zcnbridge/transaction/txn.go:37]: why can't we use core transaction completely?

Comment (Dmdv) [reply zcnbridge/transaction/txn.go:37]: It's different

**Review by dabasov — APPROVED** [review]
lgtm

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | refactor_authorizer  |
| zboxcli  | upgraded_gosdk  |
| zwalletcli  | sdk_update  |
| 0chain  | refactor_authorizer  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Pull request merged**